GuildedRoseLLCSpec/ItemParserSpec.swift

```diff
@@ -0,0 +1,59 @@
+import Quick
+import Nimble
+import GuildedRoseLLC
+import UIKit
+
+class ItemParserSpec : QuickSpec {
+    
+    override func spec() {
+        
+        describe("parse function"){
+            it("parses a properly formattted json object into a list of items") {
```

Comment:
Request for changes: Did you test the case of no items in the array? One item in the array? Did you test the case of a totally empty JSON string? What if the JSON is completely malformed (i.e. it is an object instead of a list)?

Reply:
Implemented